### ISSUE TITLE
workspace module

### ISSUE THREAD

**Issue by elit3rjmg** [prasanthrangan/hyprdots#505]
![231117_01h49m54s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/84078883/282c9f23-cdcb-4c70-9daa-b9f2f2b48b60)

The workspace bar used to show only active workspaces, how do i change it back?

**Comment by T-Crypt** [comment]
@elit3rjmg 

You need to update the configs!

`$HOME/Hyprdots/Scripts/Install.sh -r` 

Then you'll have the updated workspaces module.

or add manually to `~/.config/waybar/modules/workspace.json`

```
  "hyprland/workspaces": {
        "disable-scroll": true,
        "all-outputs": true,
        "active-only": false,
        "on-click": "activate",
        "persistent-workspaces": {
            "1": []
        }
    },
```

**Comment by kRHYME7** [comment]
Hi @T-Crypt  does having "1" on persistent-workspace make 1  persistent right? even there is no active window?

**Comment by elit3rjmg** [comment]
@kRHYME7 Yea it fixed it. Now it it only shows active workspaces.

**Comment by T-Crypt** [comment]
@kRHYME7 it only shows 1 until you have more active workspaces open.

**Comment by prasanthrangan** [comment]
resolved in dfca1e8, so closing this!
